cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/ticket/accesstoken/DefaultAccessTokenFactory.java

```diff
@@ -29,7 +29,7 @@
     /** Default instance for the ticket id generator. */
     @NotNull
     @Autowired(required = false)
-    @Qualifier("accessTokenIdGenerator")
+    @Qualifier("refreshTokenIdGenerator")
```

Comment:
Good catch. Will fix and do some renamings for consistency as I realized that the `oAuth` prefix is not necessary for access and refresh tokens.